### PR TITLE
V1.4 Minor update docs for filter feature documentation

### LINKED ISSUE #231: Since all searches are case-insensitive, perhaps you can put it at the start tgt with the multiple filter clause, because its also a feature
Since all searches are case-insensitive, perhaps you can put it at the start tgt with the multiple filter clause, because its also a feature

_Originally posted by @appleraincoat in https://github.com/AY2324S2-CS2103T-W10-1/tp/pull/226#discussion_r1550779959_

### LINKED ISSUE #232: might want to follow our standard order
might want to follow our standard order 
name, tag, housing type, remark, birthday iirc

_Originally posted by @wapisai in https://github.com/AY2324S2-CS2103T-W10-1/tp/pull/226#discussion_r1550827649_

### PR DESCRIPTION
**Pull request by jyztintan**
Fix #231, fix #232

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -159,6 +159,7 @@ The filter command in Realodex allows you to narrow down your list of clients by
 related to one of the client's attributes: name, remark, tag, birthday month, or housing type.
 This feature is particularly useful when you need to focus on a
 subset of your client database that meets certain criteria.
+Note that the search is **case-insensitive** for all input parameters.
```
Comment (appleraincoat) [inline docs/UserGuide.md:162]: 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -240,13 +215,54 @@ tag(s) specified in the input.
...
+    
+<u>Example:</u>
+
+  <a href="images/filter/filterByRemarkScreenshot.png">
+    <img src="images/filter/filterByRemarkScreenshot.png" alt="filterByRemarkScreenshot" style="width:150%">
+    </a>
+<p align="center">
+  <em> <code>filter r/eat</code> returns persons with remarks like "Eats alot" and "Likes to eat nasi lemak ..."</em>
+</p>
+
```
Comment (appleraincoat) [inline docs/UserGuide.md:259]: thanks for re-ordering!

**Review by appleraincoat — APPROVED** [review]
LGTM

**Review by UdhayaShan1 — APPROVED** [review]
LGTM thanks!

**Pull request merged**